src/server.toit

```diff
@@ -191,27 +191,32 @@ class ResponseWriter:
   write_headers status_code/int --message/string?=null:
     if body_writer_: throw "headers already written"
     has_body := status_code != STATUS_NO_CONTENT
-    write_headers_ status_code --message=message --has_body=has_body
+    write_headers_
+        status_code
+        --message=message
+        --content_length=null
+        --has_body=has_body
 
   write data:
-    write_headers_ STATUS_OK --message=null --has_body=true
+    write_headers_ STATUS_OK --message=null --content_length=data.size --has_body=true
     body_writer_.write data
 
-  write_headers_ status_code/int --message/string? --has_body/bool:
+  write_headers_ status_code/int --message/string? --content_length/int? --has_body/bool:
```

Comment:
I think it would save code to have content_length default to null here.

Reply:
keeping as is, to make callers think about the content-length.